documentation/library_guide/kernel_templates/kernel_configuration.rst

```diff
@@ -0,0 +1,39 @@
+Kernel Configuration
+####################
+
+.. code:: cpp
+
+   // Defined in header <oneapi/dpl/experimental/kernel_templates>
+
+   template <std::uint16_t DataPerWorkItem,
+             std::uint16_t WorkGroupSize,
+             typename KernelName = oneapi::dpl::execution::DefaultKernelName>
+   struct kernel_param;
+
+Each Kernel Template is supplied by a `kernel_param` object, generic structure for configuring a kernel.
```

Comment:
Is there a reason Kernel Template is capitalized outside of headers? If not, please change it. My suggestions:

Each kernel template is supplied by a `kernel_param` object, a generic structure for configuring a kernel. The supported and the most performant values depend on the kernel template itself, device capabilities, the data type, and the data size.

Reply:
The names in the text body are now not capitilized, the suggestion has been applied.

Reply:
Thank you.